config/routes.rb

```diff
@@ -42,4 +43,6 @@
   get "expired_token", to: "pages#expired_token"
 
   resources :invitations, only: [:new, :create]
+
+  mount StripeEvent::Engine, at: '/stripe_event'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.